src/loaders/configure/configure.js

```diff
@@ -25,7 +25,7 @@ export function configure(agentIdentifier, opts = {}, loaderType) {
     setRuntime(agentIdentifier, runtime)
 
 
-    setAPI(agentIdentifier, api)
+    setAPI(agentIdentifier, api, forceDrain)
```

Comment:
Some browser tests need the API to drain without loading any features.  This forces it to drain regardless of other things in the drain "registry"